utils/setup/install_deps.sh

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env bash
 
 function check_python_version_gte_3_7 {
-    echo "Checking for >=python3.7"
+    echo "Checking for >=python3.8"
     # running through current minor versions
-    hash python3.7 2>/dev/null \
+    hash python3.8 2>/dev/null \
     || hash python3.8 2>/dev/null \
     || hash python3.9 2>/dev/null;
 }
 
 function install_python_3_7 {
-    echo "Installing python3.7"
+    echo "Installing python3.8"
     sudo apt-get install $1 software-properties-common
     sudo add-apt-repository $1 ppa:deadsnakes/ppa
-    sudo apt-get install $1 python3.7 python3.7-tk python3.7-venv
+    sudo apt-get install $1 python3.8 python3.8-tk python3.8-venv
```

Comment:
Do you know why we need python Tkinter? It is not in most other requirements.

Reply:
I'm not sure. It seems it's been there for a while.